dev/customer_testing/pubspec.yaml

```diff
@@ -19,4 +19,8 @@ dependencies:
   string_scanner: 1.1.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   term_glyph: 1.2.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
 
-# PUBSPEC CHECKSUM: e9e7
+dev_dependencies:
+  # See packages/flutter_test/pubspec.yaml for why we're pinning this version.
+  test: 1.16.4
+
+# PUBSPEC CHECKSUM: aa2d
```

Comment:
I don't understand how you managed to get just this file updated and not all of the `pubspec.yaml`s, nor how you managed to pin `test` here without updating the pin list in the `update_packages.dart` file...

Reply:
I didn't use update-packages correctly originally, ran it with the force-upgrade